packages/fast-components-styles-msft/src/utilities/color/accent-foreground.spec.ts

```diff
@@ -26,8 +26,13 @@ describe("accentForeground", (): void => {
     });
 
     test("should accept a function that resolves a background swatch", (): void => {
-        expect(typeof accentForegroundRest(() => "#FFF")).toBe("function");
-        expect(accentForegroundRest(() => "#000")({} as DesignSystem)).toBe(
+        // TODO: #1473
```

Comment:
Could you add a link to the issue and/or description along with the issue number?